addons/web_tour/views/tour_views.xml

```diff
@@ -1,46 +1,87 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-        <record id="edit_tour_form" model="ir.ui.view">
-            <field name="model">web_tour.tour</field>
-            <field name="arch" type="xml">
-                <form create="0" edit="0">
-                    <sheet>
+    <record id="tour_form" model="ir.ui.view">
+        <field name="model">web_tour.tour</field>
+        <field name="arch" type="xml">
+            <form create="0">
+                <header>
+                    <div>
+                        <field name="name" widget="tour_start_widget"/>
+                    </div>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <h1><field class="text-break" options="{'line_breaks': False}" widget="text" name="name" placeholder="e.g. My_Tour"/></h1>
+                    </div>
+                    <group>
                         <group>
-                            <group>
-                                <field name="name"/>
-                            </group>
-                            <group>
-                                <field name="user_id"/>
-                            </group>
+                            <field name="sequence"/>
+                            <field name="url"/>
                         </group>
-                    </sheet>
-                </form>
-            </field>
-        </record>
+                        <group>
+                            <field name="custom" readonly="1"/>
+                            <field name="sharing_url" widget="CopyClipboardURL"/>
+                        </group>
+                        <notebook>
+                            <page name="steps" string="Steps" invisible="not custom">
+                                <field name="step_ids">
+                                    <list>
+                                        <field name="sequence" widget="handle"/>
+                                        <field name="trigger"/>
+                                        <field name="run"/>
+                                        <field name="content"/>
+                                    </list>
+                                </field>
+                            </page>
+                        </notebook>
+                        <field colspan="2" name="rainbow_man_message" widget="html" placeholder="Rainbow Man Message..."/>
```

Comment:
why the widget html? Either it’s already an html field and it’s not needed, either it’s a text field and you create an xss vector :monocle_face:

Reply:
Old vestige of previous try. It's removed, thx! :D